### PR TITLE
fix(fastboot): verify not fastboot prior accessing this.element

### PR DESCRIPTION
**Pull request by sarbbottam**
guard every this.element access

#14 got merged by mistake.

### CODE REVIEW

**Comment by nlfurniss** [comment]
Dinesh said you need to add `ember-cli-fastboot` as a dep for the service to be available

File: packages/ember-self-focused/package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "ember-ajax": "^3.0.0",
     "ember-cli": "~3.2.0",
     "ember-cli-dependency-checker": "^2.0.0",
+    "ember-cli-fastboot": "^2.0.4",
```
Comment (nlfurniss) [inline packages/ember-self-focused/package.json]: Shouldn't this be regular dependencies?

Comment (sarbbottam) [reply packages/ember-self-focused/package.json]: Hmm, not sure. Dinesh suggested it to a `devDependency` also I used `ember-cli i ember-cli-fastboot` which added it as `devDependency`.

Comment (nlfurniss) [reply packages/ember-self-focused/package.json]: Right you are, sorry I missed that

File: packages/ember-self-focused/addon/components/self-focused.js [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+
   init() {
     this._super(...arguments);
     this.get('focusManager').updateIsFirstRender();
   },
 
   didInsertElement() {
     this._super(...arguments);
-    this.get('focusManager').setNodeToBeFocused(this.element, 'insert');
+    if (!this.get('isFastBoot')) {
```
Comment (dnalagatla) [inline packages/ember-self-focused/addon/components/self-focused.js]: We don't need a check here as fastboot library doesn't invoke `didInsetElement` hook in node process.

Comment (sarbbottam) [reply packages/ember-self-focused/addon/components/self-focused.js]: Updated.

**Comment by nlfurniss** [comment]
lgtm!

**Comment by nlfurniss** [comment]
lgtm

**Comment by sarbbottam** [comment]
Thank you @nlfurniss @dnalagatla for the review. Will release the patch soon.

**Pull request merged**